pywinauto/application.py

```diff
@@ -1247,6 +1248,33 @@ def kill(self):
     # Non PEP-8 aliases
     kill_ = Kill_ = kill
 
+    def is_process_running(self):
+        """
+        Checks that process is running.
+        Can be called before start/connect.

```

Comment:
In this case 3 Codacy warnings are correct. See [Multi-line Docstrings in PEP-257](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings).